app/src/main/java/ca/bc/gov/bchealth/ui/login/BcscAuthViewModel.kt

```diff
@@ -184,7 +184,6 @@ class BcscAuthViewModel @Inject constructor(
             }
             _authStatus.update {
                 it.copy(
-                    showLoading = false,
                     userName = userName,
```

Comment:
showLoading = false, is required here based on the condition used in the observer.